fr.inria.Coccinelle.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: fr.inria.Coccinelle
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ocaml
+command: spatch
+
+finish-args:
+  - '--filesystem=host'
+
+build-options:
+  append-path: /usr/lib/sdk/ocaml/bin
+
+modules:
+  - name: coccinelle
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/coccinelle/coccinelle.git
+        tag: '1.2'
+        commit: 969cb12e9e9b7d4f42c2ff15296fd927f1ba63af
+
+  - name: metainfo
```

Comment:
this doesn't need to be separate module.

Reply:
Thanks. Moved to post-install.